src/bionics.cpp

```diff
@@ -723,6 +731,33 @@ void player::process_bionic( int b )
     } else if (bio.id == "bio_hydraulics") {
         // Sound of hissing hydraulic muscle! (not quite as loud as a car horn)
         sounds::sound( pos(), 19, _("HISISSS!"));
+    } else if( bio.id == "bio_cable" ) {
+        const std::vector<item*> cables = items_with( []( const item &it ) {
+            return it.active && it.has_flag( "CABLE_SPOOL" );
+        });
+
+        constexpr int battery_per_power = 10;
+        int wants_power_amt = battery_per_power;
+        for( const item *cable : cables ) {
+            const auto &target = cable->get_cable_target();
+            vehicle *veh = g->m.veh_at( target );
+            if( veh == nullptr ) {
+                continue;
+            }
+
+            int got_power = wants_power_amt - veh->discharge_battery( wants_power_amt );
+            wants_power_amt -= got_power;
```

Comment:
If I get it right:
`wants_power_amt = wants_power_amt - wants_power_amt + veh->discharge_battery( wants_power_amt );`
or:
`wants_power_amt = veh->discharge_battery( wants_power_amt );`
Was that really intended?